src/wolfio.c

```diff
@@ -1078,6 +1078,13 @@ int wolfIO_HttpProcessResponse(int sfd, const char** appStrList,
                 start[len] = 0;
             }
             else {
+            #ifdef WOLFSSL_NONBLOCK_OCSP
+                result = wolfSSL_LastError();
```

Comment:
Does LastError() return errno on most systems now which wouldn't match SOCKET_EWOULDBLOCK?

Reply:
I'm open to change this, but this is copy/paste from the existing `EmbedReceive`, `EmbedSend`, `EmbedReceiveFrom` and `EmbedSendTo`.